readme-eng.md

```diff
@@ -1,222 +1,321 @@
+  * `+d` indicates the bonus to damage of a melee attack.
+  * Either parameter can be negative.
+  * Some heavy armor will omit `+d` and only display something like `(-2)` indicating a negative bonus to accuracy.
+* `[a,+b]` or `[+b]`
+  * Used on armor.
+  * `a` indicates the natural increase to AC of the piece of armor.
+  * `+b` indicates the magical bonus.
+* `(+p)` or `(+p Noun)`
+  * Indicates a modifier to item's unique effect.
+  * For example, in case of a *Ring of Damage* it indicate the bonus to damage.
+  * `Noun` is used in weapons and armors, telling what the bonus is for.
+* `(xN)`
+  * Notation used in projectile weapons.
+  * `N` is the damage multiplier for projectiles.
+
+### Example 1: Dagger (2d4) (+10,+15) [+2] (+3)
+
+ * `(2d4)` - Base damage dice
+ * `(+10,+15)` - +10 accuracy bonus, +15 damage bonus
+ * `[+2]` - Magically increases AC by +2
+ * `(+3)` - Power of the special ability of this dagger, which can be known after \*Identifying* (or researching) it.
+
+### Example 2: Elven Chainmail (-2) [14,+12] (+3 Stealth)
+
+ * `(-2)` - -2 accuracy bonus
+ * `[14,+12]` - 14 base AC bonus, +12 magical AC bonus
+ * `(+3 Stealth)` - Increases Stealth stat by 3
+
+
+# Advanced play
+
+## Macros
+
+You will often find yourself repeating the same combination of keypresses multiple times during your game, things like
+shooting at the closest target with your bow or casting a bunch of buffs on yourself at once. Macros allow you to
+do that with one keypress.
+
+* Press `@` to open *Interact with Macros* page.
+* You have two options here:
+  * **Macros** are a number of keys pressed in sequence, that will trigger some action. For example setting it to `lpo`
+    would require you to press keys `l`, `p` and `o` in quick succession to trigger the effect.
+  * **Keymaps** define a new behavior for a single keypress. It's advised to use keys which normally have no mapping (
+    the easiest way to find those is to just keep pressing keys on your keyboard and see which one shows gibberish).
+* Now to create your new macro/keymap:
+  * Press the key for either *Create a macro* or *Create a keymap*
+  * Press the key sequence/key you want to assign a new action
+  * Type the sequence of actions you want to trigger
 
-The score server is here:
-  http://hengband.sourceforge.jp/en/score.html
+## Inscriptions
 
+Items can be inscribed to add notes but also to trigger special behavior. Below is a list of special inscriptions:
+
+ * `=g`
+    * Will auto-pick these items from the floor.
+    * Most convenient on ammunition or items you throw.
+    * Takes priority over Auto-Picker - even if it'd normally destroy the object, with this inscription the item will
+      be picked up
+ * `#<NAME>`
+    * Displays the name as if it was an artifact name of the item.
+    * For example: **a Broad Sword (2d6) (+0,+0)** inscribed with `#of Yeeks` will be displayed as **a Broad Sword of Yeeks (2d6) (+0,+0)**
```

Comment:
I wrote (2d6) in my previous point, but in fact the normal broad sword die roll is (2d5), so it would be better to match that.

Reply:
I play Hengbang very casually, I wouldn't know a difference between the correct and incorrect damage die :)